discord/message.py

```diff
@@ -1114,3 +1148,30 @@ async def ack(self):
         if state.is_bot:
             raise ClientException('Must not be a bot account to ack messages.')
         return await state.http.ack_message(self.channel.id, self.id)
+
+    async def reply(self, content=None, **kwargs):
```

Comment:
Might be worth adding a shorthand to `Context`?

Reply:
This can be marked as resolved

Reply:
Already done